example-model/src/main/java/uk/gov/gchq/palisade/example/client/ExampleSimpleClient.java

```diff
@@ -24,40 +25,31 @@
 import uk.gov.gchq.palisade.example.common.ExampleUsers;
 import uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee;
 import uk.gov.gchq.palisade.example.util.ExampleFileUtil;
+import uk.gov.gchq.palisade.example.web.DataClient;
+import uk.gov.gchq.palisade.example.web.PalisadeClient;
 
+import java.io.IOException;
 import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.stream.Stream;
 
-public class ExampleSimpleClient<T> extends SimpleClient<Employee> {
+public class ExampleSimpleClient extends SimpleClient<Employee> {
     private static final Logger LOGGER = LoggerFactory.getLogger(ExampleSimpleClient.class);
     private static final String RESOURCE_TYPE = "employee";
 
-
-    public ExampleSimpleClient() {
-        super(new AvroSerialiser<>(Employee.class));
+    public ExampleSimpleClient(final PalisadeClient palisadeClient, final DataClient dataClient, final EurekaClient eurekaClient) {
+        super(new AvroSerialiser<>(Employee.class), palisadeClient, dataClient, eurekaClient);
     }
 
-    public static void main(final String[] args) {
-        if (args.length == 3) {
-
-            String userId = args[0];
-            String filename = args[1];
-            String purpose = args[2];
-            User user = ExampleUsers.getUser(userId);
-            LOGGER.info(String.format("%s is reading the Employee file %s with a purpose of %s", user.getUserId().toString(), filename, purpose));
-            final Stream<Employee> results = new ExampleSimpleClient().read(filename, user.getUserId().getId(), purpose);
-            LOGGER.info(user.getUserId().toString() + " got back: ");
-            results.map(Object::toString).forEach(LOGGER::info);
-
-        } else {
-            System.out.printf("Usage: %s userId resource purpose\n\n", ExampleSimpleClient.class.getSimpleName());
-            System.out.println("userId\t\t the unique id of the user making this query");
-            System.out.println("resource\t the name of the resource being requested");
-            System.out.println("purpose\t\t purpose for accessing the resource");
-        }
+    public void run(final String userId, final String filename, final String purpose) throws IOException, URISyntaxException {
+        User user = ExampleUsers.getUser(userId);
+        LOGGER.info(String.format("%s is reading the Employee file %s with a purpose of %s", user.getUserId().toString(), filename, purpose));
```

Comment:
update to remove the string.format method and so it uses the same format used everywhere else